tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -530,15 +531,16 @@ describe(__filename, () => {
       dispatchAddon();
       dispatchAddonReviews();
       const root = render();
-      expect(root.find(Paginate))
+      expect(shallow(root.find('.AddonReviewList-reviews').prop('footer')))
```

Comment:
I'm a bit concerned that we're not verifying that the `footer` property is in fact a `Paginator`. Is there a simple way to add that to the test? The same goes for the other tests which are verifying that the `Paginator` is correctly configured.

Reply:
Yes, I think we could do better here. First, introducing a variable named correctly and we can probably verify the component passed to this prop.